### PR TITLE
reorder coutry and home page and add text edits

### PR DESCRIPTION
**Pull request by WKocur**
Sorry, I really should have broken this into more commits :( The main things I've done:

- Text edits
- Addition of citation and downloads section to homepage
- Change ordering of sections and heading hierarchies
- Modal now takes a string (html) not array or object
- Some refactoring in the production of modal content

I wouldn't worry too much about the first 3 above - Osgur is checking staging and should spot these.

This doc records the changes: https://docs.google.com/document/d/12t6rZ4JhR-GcAupsIGbdoALVK8eVseZSCjcG9mP5-oU/edit#

Not sure who is best to review this, but from the git blame, looks like @stanleypliu and @lucacug were working on it recently?

### CODE REVIEW

File: app/views/countries/partials/_habitat-representation.html.erb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <section class="container section-padding--top">
   <div class="flex">
-    <h2>
+    <h3>
```
Comment (stanleypliu) [inline app/views/countries/partials/_habitat-representation.html.erb:3]: Not that this is a big issue, but I feel that this should be an h2 as it's representing a primary section of the page.

**Comment by stanleypliu** [comment]
Screenshot 2020-06-26 at 10 15 55

Are these links meant to be live too?

**Comment by WKocur** [comment]
@stanleypliu How far did you get with this PR?

Also I've made a few more commits, but have done smaller ones this time, so hopefully they'll be easier to review.

File: app/views/countries/partials/_habitat-representation.html.erb (hunk 1 above)
Comment (WKocur) [reply app/views/countries/partials/_habitat-representation.html.erb:3]: Yes, this is now a subsection and the grey backgrounds do need updating. This is planned to be looked at, but is being left as is for the moment. I suspect it's one to go through Charlotte, but is being left until after launch.

**Comment by stanleypliu** [comment]
Everything looks good, I'm happy to merge these changes in.

**Pull request merged**